huxley/www/static/js/huxley/components/AdvisorProfileView.js

```diff
@@ -243,9 +325,105 @@ var AdvisorProfileView = React.createClass({
           </div>
           <div className="tablemenu footer">
           </div>
+          <Button
+            color="green"
+            loading={this.state.loading}
+            type="submit">
+            Save
+          </Button>
+          <span className="help-text"><em> Remember to save any changes!</em></span>
         </form>
       </InnerView>
     );
   },
+
+  renderError: function(field) {
+    if (this.state.errors[field]) {
+      return (
+        <label className="hint error">
+          {this.state.errors[field]}
+        </label>
+      );
+    }
+
+    if (this.state.errors.school &&
+        this.state.errors.school[field]) {
+      return (
+        <label className="hint error">
+          {this.state.errors.school[field]}
+        </label>
+      );
+    }
+
+    return null;
+  },
+
+  _handleChange: function(fieldName, event) {
```

Comment:
This is a general purpose handle function that could be used to clean up handlers elsewhere as well. The alternative to the conditional here would be to modify custom components (like PhoneInput) to pass the entire event object to the onChange prop instead of just a value.